volatility3/framework/plugins/linux/mountinfo.py

```diff
@@ -0,0 +1,220 @@
+                                            "with similar output format to the older /proc/[pid]/mounts or the "
+                                            "user-land command 'mount -l'.",
+                                            optional=True,
+                                            default=False),
+        ]
+
+    @classmethod
+    def _do_get_path(cls, mnt, fs_root) -> Union[None, str]:
+        """It mimics the Linux kernel prepend_path function."""
+        vfsmnt = mnt.mnt
+        dentry = vfsmnt.get_mnt_root()
+
+        path_reversed = []
+        while dentry != fs_root.dentry or vfsmnt.vol.offset != fs_root.mnt:
+            if dentry == vfsmnt.get_mnt_root() or dentry.is_root():
+                parent = mnt.get_mnt_parent().dereference()
+                # Escaped?
+                if dentry != vfsmnt.get_mnt_root():
+                    return None
+
+                # Global root?
+                if mnt.vol.offset != parent.vol.offset:
+                    dentry = mnt.get_mnt_mountpoint()
+                    mnt = parent
+                    vfsmnt = mnt.mnt
+                    continue
+
+                return None
+
+            parent = dentry.d_parent
+            dname = dentry.d_name.name_as_str()
+            path_reversed.append(dname.strip("/"))
+            dentry = parent
+
+        path = "/" + "/".join(reversed(path_reversed))
+        return path
+
+    @classmethod
+    def get_mountinfo(cls, mnt, task) -> Union[None, Tuple[int, int, str, str, str, List[str],
+                                                           List[str], str, str, List[str]]]:
+        """Extract various information about a mount point.
+        It mimics the Linux kernel show_mountinfo function.
+        """
+        mnt_root = mnt.get_mnt_root()
+        if not mnt_root:
+            return None
+
+        path_root = cls._do_get_path(mnt, task.fs.root)
+        if path_root is None:
+            return None
+
+        mnt_root_path = mnt_root.path()
+        superblock = mnt.get_mnt_sb()
+
+        mnt_id: int = mnt.mnt_id
+        parent_id: int = mnt.mnt_parent.mnt_id
+
+        st_dev = f"{superblock.major}:{superblock.minor}"
+
+        mnt_opts: List[str] = []
+        mnt_opts.append(mnt.get_flags_access())
+        mnt_opts.extend(mnt.get_flags_opts())
+
+        # Tagged fields
+        fields: List[str] = []
+        if mnt.is_shared():
+            fields.append(f"shared:{mnt.mnt_group_id}")
+
+        if mnt.is_slave():
+            master = mnt.mnt_master.mnt_group_id
+            fields.append(f"master:{master}")
+            dominating_id = mnt.get_dominating_id(task.fs.root)
+            if dominating_id and dominating_id != master:
+                fields.append(f"propagate_from:{dominating_id}")
+
+        if mnt.is_unbindable():
+            fields.append("unbindable")
+
+        mnt_type = superblock.get_type()
+
+        devname = mnt.get_devname()
+        if not devname:
+            devname = "none"
```

Comment:
The `devname` variable is finally returned and appears to the user. If possible, I think we have a better direction than the `none` string. (like `UnreadableValue` or `UnparsableValue`)
```suggestion
        devname = mnt.get_devname()
        if not devname:
            devname = renderers.UnreadableValue()
```